src/logger.py

```diff
@@ -44,15 +45,15 @@ def emit(self, record: logging.LogRecord) -> None:
 console_handler.setLevel(logging.DEBUG)
 
 supabase_handler = SupabaseHandler()
-supabase_handler.setLevel(logging.INFO) 
+supabase_handler.setLevel(logging.INFO)
 
 
 # create a formatter for the console handler
-console_formatter = logging.Formatter('[%(levelname)s]: %(message)s')
+console_formatter = logging.Formatter("[%(levelname)s]: %(message)s")
```

Comment:
is there any specific reason you switch from single to double quotes? Was that your formatter?

Following the docs, there is literally no difference, so the change is completely unnecessary. The only thing that is important is to be consistent and that was definitely screwed up by your formatter....

Reply:
This is the default of the formatter, I tried to change it to single quotes but it did not work. I really tried

Reply:
can you push the config for your formatter into the .vscode folder? Then it should be applied for me automatically.

I don't really care which quotes, but I will fall back to single as I use them in all languages all the time for years.